### PR TITLE
replace tensorboard checks with helper function

### PR DESCRIPTION
**Pull request by winglian**
# Description

There's a lot of duplicate code and boilerplate for checking tensorboard log values. This helper function should make it much easier to check values from e2e trainings without a lot of extra code.

### CODE REVIEW

File: tests/e2e/multigpu/test_eval.py [hunk 1]
```diff
@@ -7,13 +7,11 @@
 
 import yaml
 from accelerate.test_utils import execute_subprocess_async
-from tbparse import SummaryReader
 from transformers.testing_utils import get_torch_dist_unique_port
+from utils.tensorboard_ import check_tensorboard
```
Comment (djsaunde) [inline tests/e2e/multigpu/test_eval.py]: Why is `utils` sorted as a 3rd party package?

**Review by djsaunde — APPROVED** [review]
Seems like an easy win, just a nit about the `isort` order of the `utils` module.

File: tests/e2e/multigpu/test_eval.py (hunk 1 above)
Comment (winglian) [reply tests/e2e/multigpu/test_eval.py]: ah, conflict in utils under axolotl and utils under tests maybe? let me see if I can make that work properly

Comment (winglian) [reply tests/e2e/multigpu/test_eval.py]: moving the helper fn and converting to a relative import seems to make it more obvious now.

**Pull request merged**